scripts/deploy-backend.sh

```diff
@@ -106,7 +106,8 @@ update_backend()
 
     # Add network policy to allow frontend to access backend
     cf add-network-policy "$CGAPPNAME_FRONTEND" "$CGAPPNAME_BACKEND" --protocol tcp --port 8080
-
+    cf add-network-policy "$CGAPPNAME_BACKEND" tdp-clamav-nginx-$env --protocol tcp --port 9000
+    
```

Comment:
@raftmsohani is there more than one tdp-clamav-nginx? I thought there was only one in staging.